packages/server/src/Store.ts

```diff
@@ -0,0 +1,7 @@
+export class Store {
+  rooms: string[];
```

Comment:
bot sure but maybe it's a good idea to use Set since duplicates can't be possible here
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Set

Reply:
This is very temporary, it will eventually be something that can be serialisable object, that ends up in db. But ok, I can change that, it's never going to have a conflict anyway now.